api/configuration_services.proto

```diff
@@ -57,6 +58,89 @@ message GetPhyVersionsResponse {
   repeated VersionInfo version_info = 2;
 }
 
+message ListBandsRequest {
+  // Optional Band ID to filter the results.
+  // If unused, all supported Bands and their versions are returned.
+  string band_id = 1;
```

Comment:
We should also provide a `PHYVersion` filter, with the same semantics (applied if present).